### DISCUSSION TITLE
game request

### DISCUSSION THREAD

**Discussion by nonoguynono** [QuiteAFancyEmerald/InvisiProxy#296] [Ideas]
can u add the 1st & 2nd run

**Comment by T0XlcGamer** [comment]
I would love it if you added Mike Tyson and super punch out

**Comment by T0XlcGamer** [comment]
Alos there is a hole website called www.playretrogames.com if you need more games to add.

**Reply by T0XlcGamer** [reply]
Um it's not working so try this 
https://www.playretrogames.com/super-nintendo/all
